src/components/VideoList/index.jsx

```diff
@@ -20,7 +20,7 @@ function VideoList({ youtubeVideId }) {
   }, []);
 
   return (
-    <Link to={`/watch?${video.youtubeVideoId}`}>
+    <Link state={{ video }} to={`/watch?${video.youtubeVideoId}`}>
```

Comment:
```suggestion
    <Link to={`/watch?${video.youtubeVideoId}` state={{ video }} }>
```
to 속성이 앞으로 나와주는게 좋을 것 같습니다!